source/ore/gradle-deploy.rst

```diff
@@ -0,0 +1,86 @@
+==================
+Deploy from Gradle
+==================
+
+SpongeGradle_ makes it easy for plugin developers to deploy new versions of their plugin directly to Ore without even
+needing to open a browser. Once you have created a project on Ore and have published at least one version of your
+plugin, you can begin to deploy new versions of your plugin from your build script.
+
+.. tip::
+    For more info on how to get started with Gradle and Sponge, check out :doc:`/plugin/project/gradle`.
+
+Configuring the signing plugin
+==============================
+
+Like all Ore uploads, the artifacts you publish from Gradle must be signed with the PGP private key that corresponds to
+the public key you've added to your account in :doc:`/ore/publish`. Luckily, this is easily accomplished with Gradle as
+it comes with a core-plugin that does just this. In order to use the ``oreDeploy`` task in Gradle, *you must*
+configure the signing plugin as it depends on this specific plugin.
+
+For more information on how to configure the signing plugin, please refer to the Gradle User Guide entry:
+`The Signing Plugin <https://docs.gradle.org/current/userguide/signing_plugin.html>`__.
+
```

Comment:
It might be worth mentioning (Not saying you should, but it's an idea) that you can use Script Parameters such as 
`-Psigning.keyId=<KEY ID> -Psigning.password=<Password> -Psigning.secretKeyRingFile=<Secring.gpg>`
to keep your gpg info private =D (It took me a while to find that one myself, but i'm not exactly a pro lol)

Reply:
This would be useful, especially when automating the deployment from a CI or something. 👍

Reply:
Fixed